.goreleaser.yml

```diff
@@ -0,0 +1,132 @@
+before:
+  hooks:
+    - go version | grep --quiet "go1\.15\.2" || echo "Go binary version must be 1.15.2"
```

Comment:
1.15.3 is out now. Is there anyway for use to enforce the latest available version of 1.15?

Reply:
Yeah with some fancier `grep`ing. I was thinking of pinning this version to a patch for pure reproducibility reasons though.